easybuild/easyconfigs/l/librosa/librosa-0.7.2-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'librosa'
+version = '0.7.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://librosa.github.io'
+description = "Python module for audio and music processing"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('FFmpeg', '4.2.1'),
+    ('matplotlib', '3.1.1', versionsuffix),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('numba', '0.47.0', versionsuffix),
+]
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('audioread', '2.1.8', {
+        'checksums': ['073904fabc842881e07bd3e4a5776623535562f70b1655b635d22886168dd168'],
+    }),
+    ('SoundFile', '0.10.3.post1', {
```

Comment:
I think this depends on `libsndfile`.

Reply:
Yeah, see https://pypi.org/project/SoundFile/: 
SoundFile depends on the Python packages CFFI and NumPy, and the system library libsndfile.

Reply:
Ah, we have this installed system-wide, so that's possible...

Let's see if `generoso` catches this.

Reply:
Yep, `import soundfile` fails with `OSError: sndfile library not found` if the dep is not there \o/